diff --git a/src/Controller/PosOrderItemAutoComplete.php b/src/Controller/PosOrderItemAutoComplete.php
index e41ab28..4985e68 100644
--- a/src/Controller/PosOrderItemAutoComplete.php
+++ b/src/Controller/PosOrderItemAutoComplete.php
@@ -2,6 +2,7 @@
 
 namespace Drupal\commerce_pos\Controller;
 
+use Drupal\commerce_pos\Entity\Register;
 use Drupal\Core\Controller\ControllerBase;
 use Drupal\Core\Entity\EntityTypeManagerInterface;
 use Drupal\Core\Render\RendererInterface;
@@ -109,7 +110,7 @@ class PosOrderItemAutoComplete extends ControllerBase {
    *   The query search result.
    */
   public function searchQueryString($string, $count) {
-    $register = \Drupal::service('commerce_pos.current_register')->get();
+    $register = Register::load(1);
     if ($register) {
       $store_id = $register->getStoreId();
 
